application-task-ui/src/main/resources/TaskManager/GanttMacro/Macro.xml

```diff
@@ -0,0 +1,2059 @@
+          progressChangeObserver.disconnect();
+        }
+        progressChangeObserver = new MutationObserver(() =&gt; {popupNumber.innerHTML = bar.compute_progress();});
+        progressChangeObserver.observe(bar.$bar_progress, {attributes:true});
+      }
+      function hidePopup() {
+        if (is_resizing_progress) return;
+        popup.style.visibility = 'hidden';
+        if(progressChangeObserver) {
+          progressChangeObserver.disconnect();
+        }
+      }
+      jQuery('.handle.progress', ganttDiv).on({
+        mousedown: function(e) {
+          let bar = e.delegateTarget.parentElement.parentElement;
+          showPopup(bar);
+          is_resizing_progress = true;
+        }
+      });
+      jQuery(ganttDiv).on({
+        mouseup: function(e) {
+          is_resizing_progress = false;
+          hidePopup();
+        }
+      });
+
+      jQuery('.bar-wrapper', ganttDiv).on({
+        mouseenter: function(e) {
+          let bar = e.currentTarget;
+          if(!_hoverTimer) {
+            _hoverTimer = setTimeout(() =&gt; {
+              showPopup(bar);
+            }, 1000);
+          }
+        },
+        mouseleave: function(e) {
+          clearTimeout(_hoverTimer);
+          _hoverTimer = null;
+          hidePopup();
+        },
+        mousedown: function(e) {
+          clearTimeout(_hoverTimer);
+          _hoverTimer = null;
+          hidePopup();
+        }
+      });
+    }
+    function addStyles(svgElem) {
+      var defs = document.createElementNS('http://www.w3.org/2000/svg', 'defs');
+      defs.innerHTML += `
+    &lt;linearGradient id="fadeGrad" fill="white"&gt;
+      &lt;stop offset="30%" stop-color="white" stop-opacity="1" /&gt;
+      &lt;stop offset="60%" stop-color="white" stop-opacity=".4" /&gt;
+      &lt;stop offset="100%" stop-color="white" stop-opacity="0" /&gt;
+    &lt;/linearGradient&gt;
+    &lt;mask id="fademask" maskContentUnits="objectBoundingBox"&gt;
+      &lt;rect width="1" height="1" fill="url(#fadeGrad)"/&gt;
+    &lt;/mask&gt;
+
+    &lt;filter id="rounded-corners" x="-2.5%" width="105%" y="-5%" height="110%"&gt;
+      &lt;feFlood flood-color="rgba(255,60,60)"/&gt;
+      &lt;feGaussianBlur stdDeviation="4"/&gt;
+      &lt;feComponentTransfer&gt;
+        &lt;feFuncA type="table"tableValues="0 0 0 1"/&gt;
+      &lt;/feComponentTransfer&gt;
+      &lt;feComponentTransfer&gt;
+        &lt;feFuncA type="table"tableValues="0 1 1 1 1 1 1 1"/&gt;
+      &lt;/feComponentTransfer&gt;
+      &lt;feComposite operator="over" in="SourceGraphic"/&gt;
+    &lt;/filter&gt;
+    `;
+      svgElem.setAttribute('height', svgElem.getAttribute('height') - 100);
+      svgElem.appendChild(defs);
+    }
+    Gantt.prototype['_render'] = Gantt.prototype['render'];
+    Gantt.prototype['render'] = function() {
+      const result = this._render();
+      addStyles(this.$svg);
+      setupProgressPopup(this);
+      return result;
+    };
+  }
+  // Fix hour text display
```

Comment:
I'll elaborate. What exactly are you fixing?
Also, don't forget to add a `.` at the end of comments